backends/include/dppl_sycl_device_interface.h

```diff
@@ -110,10 +110,10 @@ DPPLDevice_GetDriverInfo (__dppl_keep const DPPLSyclDeviceRef DRef);
  */
 DPPL_API
 uint32_t
-DPPLDevice_GetMaxComputeUnites (__dppl_keep const DPPLSyclDeviceRef DRef);
+DPPLDevice_GetMaxComputeUnits (__dppl_keep const DPPLSyclDeviceRef DRef);
```

Comment:
It is very dangerous to rename API function. It is better to create a new one with new name and implement the old one via the new one.

Reply:
Then mark it as deprecated. Better to print warning in stderr when old function is used.
And remove in the next release.